cluster/manifests/z-karpenter/deployment.yaml

```diff
@@ -38,7 +38,7 @@ spec:
         effect: NoSchedule
       containers:
         - name: controller
-          image: "public.ecr.aws/karpenter/controller:v0.16.1@sha256:ed1f97bc2cd67e63b45546955cf763890182f1f97586fca5e6cbd25f175b9c56"
+          image: "container-registry.zalando.net/teapot/karpenter:v0.18.0-main-3"
```

Comment:
The webhook image should also be changed. It's in the same pod template further down on line 83

Reply:
🤦  Thanks!